Library/Homebrew/utils/formatter.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/tty"
+
+module Formatter
+  module_function
+
+  def arrow(string, color: nil)
+    prefix("==>", string, color)
+  end
+
+  def headline(string, color: nil)
+    arrow("#{Tty.bold}#{string}#{Tty.reset}", color: color)
+  end
+
+  def success(string, label: nil)
+    label(label, string, :green)
+  end
+
+  def warning(string, label: nil)
+    label(label, string, :yellow)
+  end
+
+  def error(string, label: nil)
```

Comment:
I think this would be a step back, as this gets us to basically start *formatting* on the call side again, which kinda defeats the purpose of having a formatter in the first place.